graylog2-server/src/main/java/org/graylog2/rest/resources/system/GrokResource.java

```diff
@@ -142,4 +148,43 @@ public void removePattern(@PathParam("patternId") String patternId) {
             throw new javax.ws.rs.NotFoundException();
         }
     }
+
+    private GrokPatternSummary toSummary(GrokPattern grokPattern) {
```

Comment:
I don't think those helper methods belong into the resource class. Maybe add a companion class `GrokPatterns` (analogue to `Collections` or `Objects` in the JDK)?